axelor-talent/src/main/resources/i18n/messages_fr.csv

```diff
@@ -1,203 +1,203 @@
-"My traning filters",,,
-"My upcoming appraisal",,,
-"My upcoming trainings",,,
-"Name","Nom",,
-"Nb Employee",,,
-"Nb Hours",,,
-"Nb of open jobs","Nb de postes ouverts",,
-"Nb of people hired","Nb de personnes embauchées",,
-"Nb of training hours per month","Nb d'heures de formation par mois",,
-"Nb. hours per category","Nb d'heures par catégorie",,
-"Nb. hours per training","Nb d'heures par formation",,
-"Nb. of trained employee per category","Nb d'employés formés par catégorie",,
-"New appraisal",,,
-"Number of hours","Nombre d'heures",,
-"Number of registredred to the session","Nombre d'inscrits à la session",,
-"Objectives","Objectifs",,
-"On hold","En attente",,
-"Open","Ouvrir",,
-"Opened",,,
-"Other skills","Autres compétences",,
-"Photo",,,
-"Planned","Planifié",,
-"Position Status","Statut du poste",,
-"Primary Address","Adresse principale",,
-"Profile wanted","Profil recherché",,
-"Program","Programme",,
-"Proposed salary","Salaire proposé",,
-"Publication Date","Date de publication",,
-"Rating","Note",,
-"Realize",,,
-"Reason not hired","Raison de non embauche",,
-"Recruitment","Recrutement",,
-"Referred by","Référent",,
-"Register training","Inscription à la formation",,
-"Rejected","Rejeté",,
-"Requested","Demandé",,
-"Required training","Formations requises",,
-"Responsible","Responsable du recrutement",,
-"Reviewer","Responsable de l'évaluation",,
-"Salary","Salaire",,
-"Schedule Event",,,
-"Send","Envoyer",,
-"Sent",,,
-"Sequence",,,
-"Session",,,
-"Skill","Compétences",,
-"Skills","Compétences",,
-"Skills tag","Compétences",,
-"Source","Provenance",,
-"Sources",,,
-"Starting date","Prise de poste",,
-"Status","Statut",,
-"Tag skill","Compétences",,
-"Tag skills","Compétences",,
-"Technicien",,,
-"Title","Titre",,
-"To","À",,
-"To date","Date de fin",,
-"Tools",,,
-"Total",,,
-"Training","Formations","Parent menu for training App (sub app)",
-"Training Categories",,,
-"Training Category",,,
-"Training Register","Inscriptions aux formations",,
-"Training completed","Formation réalisée",,
-"Training dashboard","Tableau de bord de la formation",,
-"Training dates must be under training session date range.",,,
-"Training per category","Formations par catégories",,
-"Training register","Inscription formation",,
-"Training registers","Inscriptions aux formations",,
-"Training session","Sessions de formation",,
-"Training sessions","Sessions de formation",,
-"Training skills","Compétences formations",,
-"Trainings","Formations","To create trainings ""template"" with a name, a description, a program, skills... ",
-"Type of appraisal","Type d'évaluation",,
-"Upcoming appraisal",,,
-"Upcoming appraisals of all employees of a team",,,
-"Upcoming trainings of all employees of a team",,,
-"Work experience","Expérience professionnelle",,
-"user@mydomain.com",,,
-"value:Appraisal","Évaluation",,
-"value:Recruitment","Recrutement",,
-"value:Training","Formations",,
+key,message,comment,context
```

Comment:
Why you have removed double quotes and where are other translation csv? Take care of this kind of unnecessary changes.